### PR TITLE
Feature/2762 delta onboarding view

### PR DESCRIPTION
**Pull request by SwissMountainFrog**
The PR includes the delta onboarding information screen for EU wide tracing.

DeltaOnboardingScreen

It contains a description, an overview of the participating countries (or in case the country list is unavailable an information statement on this) as well as a data usage privacy disclaimer.

It is based on the JIRA ticket https://jira.itc.sap.com/browse/EXPOSUREAPP-2762

### CODE REVIEW

**Review by ArturFriesen — APPROVED** [review]

**Review by janetback — APPROVED** [review]
UA Reviewed

File: src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENA.xcscheme [hunk 1]
```diff
@@ -89,6 +89,8 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = "de"
```
Comment (marcussc) [inline src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENA.xcscheme:92]: Please don't :)

File: src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/DeltaOnboardingNavigationController.swift [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import Foundation
+import UIKit
+
+final class DeltaOnboardingNavigationController: ENANavigationControllerWithFooter, UINavigationControllerDelegate, DeltaOnboardingViewControllerProtocol {
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/DeltaOnboardingNavigationController.swift:21]: Very nice :)

File: src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/V15/DeltaOnboardingV15ViewModel.swift [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+				)
+			)
+		}
+	}
+
+	// MARK: - Private
+
+	private let supportedCountries: [Country]
+
+	private func buildCountryLabel() -> [DynamicCell] {
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/V15/DeltaOnboardingV15ViewModel.swift:112]:
```suggestion
	private func buildCountryCell() -> [DynamicCell] {
```

Comment (marcussc) [reply src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/V15/DeltaOnboardingV15ViewModel.swift:112]: Also why does this return an array if we only return one cell?

**Review by marcussc — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENA.xcscheme (hunk 1 above)
Comment (ArturFriesen) [reply src/xcode/ENA/ENA.xcodeproj/xcshareddata/xcschemes/ENA.xcscheme]: removed

File: src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/DeltaOnboardingNavigationController.swift (hunk 2 above)
Comment (ArturFriesen) [reply src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/DeltaOnboardingNavigationController.swift:21]: ❤️

File: src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/V15/DeltaOnboardingV15ViewModel.swift (hunk 3 above)
Comment (ArturFriesen) [reply src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/V15/DeltaOnboardingV15ViewModel.swift:112]: If supportedCountries.isEmpty there are 2 cells.

**Review by marcussc — APPROVED** [review]

**Pull request merged**